views.go

```diff
@@ -173,22 +183,30 @@ func (lk *logkeeper) createBuild(w http.ResponseWriter, r *http.Request) {
 		TaskExecution int    `json:"execution"`
 	}{}
 	if err := readJSON(r.Body, lk.opts.MaxRequestSize, &payload); err != nil {
-		lk.logErrorf(r, "bad request to create build: %s", err.Err)
+		logErrorf(ctx, "bad request to create build: %s", err.Err)
 		lk.render.WriteJSON(w, err.code, err)
 		return
 	}
 
-	id, err := model.NewBuildID(payload.Builder, payload.BuildNum)
+	recordAttributes(
+		ctx,
+		attribute.String("builder", payload.Builder),
```

Comment:
It might be helpful to prefix all the attributes with something to make them easier to find.
In evergreen I had every attribute I set prefixed with `evergreen.<attribute>`.

Reply:
should I use the same prefix for logkeeper?

Reply:
That works for me. Or `logkeeper.<whatever>`.